### PR TITLE
Search Phrase History Browsing 

### PR DESCRIPTION
**Pull request by ddiddi**
When we in the console (cli interface), now, we can press the up and down keys to browse through history of previously searched phrases. 

The tab complete feature would allow the implementation of pressing tab and completing the phrase using Spotify API in the future, making searching for songs faster and easier.

### CODE REVIEW

File: pytify/cli.py [hunk 1]
```diff
@@ -75,6 +77,8 @@ def interaction(self):
                 if search is not False:
                     self.menu(list=self.pytify.list())
 
+def autocomplete:
```
Comment (bjarneo) [inline pytify/cli.py]: Shouldn't the function have paranthesis?
def autocomplete():

And is there any reason we need this function? :)

**Comment by bjarneo** [comment]
Great idea. I like this! Thanks for your PR.

File: pytify/cli.py (hunk 1 above)
Comment (ddiddi) [inline pytify/cli.py]: Added parenthesis.

We need this function for the following reason:
- readline.parse_and_bind function expects and argument which is of the form <key>: <function>
  -An alternative would to use tab: complete config which would use the rlcompleter default function and give a strange auto complete. Try it out.
  -the auto complete function would be used later to implement the tab auto complete feature 

Great work on Pytify! :D

Comment (bjarneo) [inline pytify/cli.py]: I see. Good to know.

I've tested it, and it works great :)

Thank you!

**Comment by bjarneo** [comment]
I'll merge this PR now. And I'll release a new version to pypi later tonight. Thank you :)

**Pull request merged**